.github/workflows/release-and-pr-nginxinc-ingress.yml

```diff
@@ -0,0 +1,89 @@
+name: Produce Docker Image with nginxinc upstream support, GH Release & Create PR for Override File
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 15 * * 1-5' # M-F at 8 AM PDT
+
+jobs:
+  # Compare nginxinc/kubernetes-ingress version with sigsci-nginx-ingress-controller version
+  # If new version build and push container to docker hub
+  # Create and tag new release
+  # Update sigsci-nginxinc-value.yaml file & create PR
+  compare_versions:
+    runs-on: ubuntu-latest
+    env:
+      GH_REPO: https://api.github.com/repos/signalsciences/sigsci-nginx-ingress-controller
+      VALUES_FILE: sigsci-nginxinc-values.yaml
+    outputs:
+      value: ${{ steps.tag.outputs.NO_CHANGE }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_API_TOKEN }}
+    - name: compare versions
+      id: tag
+      env:
+        REPO: signalsciences/sigsci-nginxinc-nginx-ingress-controller
```

Comment:
could `sigsci-nginxinc-nginx-ingress-controller`  be `sigsci-nginxinc-ingress-controller`?

Reply:
done and changed name in dockerhub